tests/test_acl_commitment_v2.py

```diff
@@ -119,7 +119,7 @@ def test_create_conference(self, client, openreview_client, helpers):
                         'value': {
                         'param': {
                             'type': 'string',
-                            'regex': '(http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+)',
+                            'regex': 'https:\/\/openreview\.net\/forum\?id=.*',
```

Comment:
could you add a test trying to post a note using the urls:

https://openreview.net/pdf?id=1234
https://openreview.net/forum?id=1234&referrer=[Author Console](/group?id=ICML.cc/2024/Conference/Authors)
https://openreview.net/forum?id=1234&replyto=4567

Reply:
@celestemartinez could you add these tests cases?